contents/spotlight/startup-puzzle.md

```diff
@@ -0,0 +1,79 @@
+
+## 3. Build pricing around user segments
+
+Next, you can delve into the quantitative data by looking at how users can be segmented into different groups based on criteria such as size or industry. Puzzle used this information to structure its pricing model, for example. 
+
+“On [our pricing page](https://puzzle.io/pricing) we make this pretty clear,” says Scott. “We have a plan for formation-stage founders, a plan for startups who have raised a little funding, and a scale plan for companies which are generating revenue.
+
+Each plan also reflects the pain points a user would deal with at that stage too and addresses it with a product feature. Monthly compliance checklists in the free plan, for example, help formation-stage founders get started quickly — but give way in the scale-up plan to AI automation and new processes.
+
+> **Putting it into practice:** If you aren’t using delineated pricing plans then it can still be helpful to think of your broader [pricing principles](/handbook/engineering/feature-pricing). Ask yourself: how do these principles work for your different user segments? 
+
+## 4. Reinforce your ICP with internal teams
+
+As you grow it’s important not to drift from your ICP or to let teams’ understanding of it become corrupted. You need to constantly remind everyone in your company who you are building for and how your product meets their needs — otherwise you risk diluting your offering. 
+
+“When I joined Puzzle I was very happy to hear people talking about our ICP,” says Scott. “Then I saw there was some confusion and some of what was said was incorrect. Some people thought our ICP is a company, while others thought it was a person, and there were some different ideas floating around.”
+
+At PostHog, we’ve solved this issue with [a public handbook](/handbook) which documents [our ICP](/handbook/who-we-are-building-for) — but Puzzle took a different approach.  
+
+“We have these all-hands meetings where everybody has to put their hand up and swear that our ICP is a founder and not an accountant,” says Scott. 
+
+> **Putting it into practice:** There are lots of ways to disseminate information internally, from intranets to incorporating it into employee onboarding. Whichever route you take though, the key is consistency. 
+
+## 5. Once you start, don’t stop
+
+Once you’ve established your ICP you must constantly maintain it. You need to always be speaking to users, working to understand their concerns, and analyzing the data behind their usage. In doing so you can find new segments to target, new needs to address, and new product improvements to make.
+
+“We started looking at our biggest user drop-offs and found many ICP users were dropping off when they had to connect their bank,” says Scott. “We reached out to these users directly and talked to them. We had to try and understand their concerns.”
```

Comment:
“While we knew we had some drop-off within our onboarding funnel, we wanted to ensure that we weren’t bringing high-intent new customers into a ‘leaky bucket’. We needed to ensure that people who completed onboarding could quickly see the value of Puzzle and activate. After digging into our data in Posthog we could see that our activation rate wasn’t what we needed it to be”, says Scott. “We reached out to these users directly and talked to them. We had to try and understand their concerns.”